wiki/quest-modding.md

```diff
@@ -138,6 +142,15 @@ latest moddable version.
 
 Visit [Modding With QAVS](./quest-modding-qavs.md) to get started.
 
+## Mods Before Friday
+
+Mods Before Friday is a WebUSB-based tool that lets you mod Beat Saber from a browser
+:::warning Supported Browsers
+Some browsers aren't supported and some say they are but then corrupt your Beat Saber.
```

Comment:
Source for this?

Reply:
It is seems unlikely that using an unsupported browser will corrupt your installation. MBF uses an agent (small executable) to perform any modifications on your quest and if there are issues with WebADB, that agent will probably get corrupted before it even runs and then fail to launch, so Beat Saber should never get damaged.

(I have never seen a report of this happening so it doesn't deserve to be included)